server/cmd/mmctl/commands/team.go

```diff
@@ -336,6 +336,8 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		return errors.New("must specify one of --private or --public")
 	}
 
+	var errs error
```

Comment:
Let stick to `*multierror.Error`
```suggestion
	var errs *multierror.Error
```

Reply:
Updated